tests/args.test.js

```diff
@@ -72,4 +70,34 @@ describe("utils", () => {
       );
     });
   });
+
+  describe("getArgString", () => {
+    it("nothing", () => {
+      expect(getArgString(_getArgs(""))).toEqual("");
+    });
+
+    it("params", () => {
+      expect(getArgString(_getArgs("--jsdebugger --set-env=headless"))).toEqual(
+        "--jsdebugger=true --set-env=headless"
+      );
+    });
+
+    it("defaults", () => {
+      expect(
+        getArgString(_getArgs("--jsdebugger --default-test-path=foo/bar"))
+      ).toEqual("--jsdebugger=true foo/bar");
+    });
+
+    it("mc included", () => {
+      expect(getArgString(_getArgs("--jsdebugger --mc bazz"))).toEqual(
+        "--jsdebugger=true"
+      );
+    });
+
+    it("test path  included", () => {
+      expect(
+        getArgString(_getArgs("--jsdebugger --default-test-path=foo/bar bazz"))
+      ).toEqual("--jsdebugger=true bazz");
+    });
+  });
```

Comment:
these tests are kinda the new gold standard the others are nice... but not as useful